### DISCUSSION TITLE
how can i use php flusher and it's features such has sweetalert, toast from blade ?

### DISCUSSION THREAD

**Discussion by md-nobir-hasan** [php-flasher/php-flasher#204] [Q&A]
is:open category:Q&A I install php flasher and sweet alert. now how can i use them from blade ? pleae, reply fast because my project has deadline. thanks.

**Comment by yoeunes** [comment]
Hello @md-nobir-hasan!

To answer your question, you don’t need to modify your Blade templates to use PHPFlasher. PHPFlasher is designed to send notifications directly from your backend controllers. It automatically injects the necessary scripts and displays the notifications on the frontend using the default configuration.

Here’s how you can use it:

1. In Your Controller:

Simply add a flash message like this in your controller method:

```php
use Flasher\SweetAlert\Prime\SweetAlertFactory;

public function yourFunction(SweetAlertFactory $flasher)
{
    // Your logic here

    $flasher->success('Your success message here');

    return redirect()->back();
}
```

This will queue the notification, and upon redirecting, PHPFlasher will handle displaying it.

2. No Need to Modify Blade Templates:

Since PHPFlasher automatically injects the necessary scripts and handles the display of notifications, you don’t need to add anything to your Blade files.

3. Default Configuration:

Ensure you’re using the default configuration, which allows PHPFlasher to work out-of-the-box without additional setup on the frontend.

If you need to customize the notifications further or have any other questions, feel free to ask.

**Comment by K-Mirembe-Mercy** [comment]
Hi! After installing PHP Flasher and SweetAlert, you can use them in Blade by including the Flasher scripts in your layout (usually in your main Blade file).
Then in your controller, you can trigger a message like: flash()->success('Message here');
And it will show in your Blade view automatically.
If it’s not showing, make sure you’ve added the Flasher rendering code in your Blade file. Could you share what you’ve tried so far?
